Content.Tests/DMProject/Tests/Procs/usr_assignment.dm

```diff
@@ -0,0 +1,9 @@
+/proc/change_usr()
+	ASSERT(usr == null)
+	usr = new/mob
```

Comment:
Unit tests should not reference `/mob` (see #1142)

Set it to a `/datum` instead
```suggestion
	usr = new/datum
```